### ISSUE TITLE
ACC-270.3 (Level A)_Kyma-dashboard_Button_UI navigated using tab and arrow key.

### ISSUE THREAD

**Issue by huda866** [kyma-project/busola#3167]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium   
Affected Check Point:	ACC-270.3
Affected UI:	Button
Short text:	ACC-270.3 (Level A)_Kyma-dashboard_Button_UI navigated using tab and arrow key.

1.2 Application Details
Application Name	Kyma-dashboard
Application URL:	https://canary.cockpit.btp.int.sap/cockpit#/globalaccount/7304343e-1ce0-412b-a955-5948bab9d2c8/subaccount/e0ec1134-2770-40a2-910b-2b8e32ec8f60/subaccountoverview
User:	SSO
Technology:	SAP UI5 
UI5 Version:	1.124.1

1.3 Test Environment Details:
GitHub	https://github.com/kyma-project/busola/issues
Web browser:	Google Chrome - Version 127.0.6533.73 (Official Build) (64-bit)
Operating System:	Windows 11 Enterprise Version 22H2
Screen Reader:	JAWS 2024.2405.83 ILM 64 bit for Windows

2. Screen Flow
•	Login into the application using the URL provided.
•	URL: https://canary.cockpit.btp.int.sap/cockpit#/globalaccount/7304343e-1ce0-412b-a955-5948bab9d2c8/subaccount/e0ec1134-2770-40a2-910b-2b8e32ec8f60/subaccountoverview.
•	Activate ‘Console URL:Link to dashboard’ in overview page.
•	Activate any ‘Hint’ button or access any ‘Learn more’ link.

3. Issue 
Synopsis: UI navigated using tab and arrow key.
Observed Behavior: 
‘Search field’ tool bar elements within all ‘Code editor’ sections are navigable using both tab and arrow keys. 

Expected Behavior: 
Buttons should be navigable using tab key instead of arrow keys.
Keyboard is the prerequisite for support of assistive technologies to enable users with disabilities to read and operate software applications, hence all the UIs must be navigable using correct key commands. 

________________________________________________________________
Kindly refer to attached error and issue processing document for more details.

Regards,
Huda

**Comment by huda866** [comment]
[ACC-270.3 (Level A)_Kyma-dashboard_Button_UI navigated using tab and arrow key.docx](https://github.com/user-attachments/files/16667922/ACC-270.3.Level.A._Kyma-dashboard_Button_UI.navigated.using.tab.and.arrow.key.docx)
[Issue processing-09.2021 (1).docx](https://github.com/user-attachments/files/16667923/Issue.processing-09.2021.1.docx)
